tests/test_umath.py

```diff
@@ -231,12 +265,17 @@ def test_invalid_out(self, func_params, out):
         func_name = func_params["func_name"]
         a = dpnp.arange(10)
         numpy.testing.assert_raises(TypeError, getattr(dpnp, func_name), a, out)
+        numpy.testing.assert_raises(
```

Comment:
Could you please extend like:
```python
from numpy.testing import (
    assert_allclose,
    assert_raises,
)
```
and so replace `numpy.testing.assert_raises` with short `assert_raises`